pkgs/tools/networking/http-prompt/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, stdenv, fetchurl, buildPythonApplication, pythonPackages }:
```

Comment:
The `pkgs` parameter is unused. I also think it may be clearer to use `pythonPackages.buildPythonApplication` instead of passing it in as a separate parameter.